docs/startup/preview/index.md

```diff
@@ -527,4 +527,5 @@ Take note of the following issues with the embedded deployment feature for the 0
     * Number of Read/Writes
     * Leader reelections
     * more..
-* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
\ No newline at end of file
+* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
+* Currently you can only run a single applicaiton with a single zookeeper.
```

Comment:
Well, it still is a limitation. That's why it requires work around. Do you guys, want to add the workaround to the doc, or prefer remove it completely?